docker/docker-entrypoint.sh

```diff
@@ -96,4 +96,4 @@ fi
 # (i.e. the RUN instruction in the Dockerfile)
 #
 # shellcheck disable=SC2068
```

Comment:
It was pointed out to me that we can remove this line as the below change is exactly what shellcheck was trying to tell us to do 🤦.

Reply:
Done!